wagtailio/project_styleguide/templates/patterns/pages/search_results/search_results.html

```diff
@@ -43,4 +44,27 @@ <h1 class="hero__heading">Search results</h1>
         </div>
     </div>
 
+    {% if search_results.paginator.num_pages > 1 %}
+        <div class="grid">
+            <div class="pagination">
+                {% if search_results.has_previous %}
+                    <a href="?page={{ search_results.previous_page_number }}{% for key,value in request.GET.items %}{% if key != 'page' %}&amp;{{ key }}={{ value }}{% endif %}{% endfor %}" class="pagination__previous button button--reverse">
+                        <p class="button__text">Previous</p>
+                        <svg class="arrow" aria-hidden="true"><use xlink:href="#arrow"></use></svg>
+                    </a>
+                {% endif %}
+
+                <span class="pagination__page-count mini-meta">
+                    Page {{ search_results.number }} of {{ search_results.paginator.num_pages }}
+                </span>
+
+                {% if search_results.has_next %}
+                    <a href="?page={{ search_results.next_page_number }}{% for key,value in request.GET.items %}{% if key != 'page' %}&amp;{{ key }}={{ value }}{% endif %}{% endfor %}" class="pagination__next button">
+                        <p class="button__text">Next</p>
+                        <svg class="arrow" aria-hidden="true"><use xlink:href="#arrow"></use></svg>
```

Comment:
nitpick: ideally we want to avoid plain svg like this and use the icon template include

```suggestion
                        {% include "patterns/components/icon/icon.html" with icon="arro" %}
```

everywhere

Reply:
Looks like the desired "arrow" (`<svg class="arrow"`) is styled differently from the one in the **include** (`<svg class="icon__svg">` inside a `div`), so it'll look like this:

![image](https://user-images.githubusercontent.com/7713776/188179302-cff45255-8a0d-46d8-ad1b-3004bdd9db62.png)

Reply:
grumble grumble. fair enough.
@William-Blackie another one for FE tech debt